### DISCUSSION TITLE
Use an image instead of the extracted map

### DISCUSSION THREAD

**Discussion by RedFoxy** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#318] [Q&A] [answered]
Hi!
There is a way to use a different image, that follow the extracted map, instead the one generated by the robot, but still keeping the path and the robot moving on it?

I want to draw a better picture of my home following the one generated by the map extractor, without all the errors and the hidden spots that my vacuum can't scan, but I don't want loose to see the vacuum postion on the map and its path when it work, there is a way to do it?

![image](https://user-images.githubusercontent.com/5691779/178486001-12c470a0-27ee-43db-a1b5-c7d1855321c1.png)

**Comment by PiotrMachowski** [comment, marked as answer]
No, it's not possible at this moment: #39
